packages/client/src/generation/TSClient/PrismaClient.ts

```diff
@@ -100,6 +100,29 @@ function executeRawDefinition(this: PrismaClientClass) {
   $executeRawUnsafe<T = unknown>(query: string, ...values: any[]): PrismaPromise<number>;`
 }
 
+function runCommandRawDefinition(this: PrismaClientClass) {
+  // we do not generate `$executeRaw...` definitions if not supported
+  if (!this.dmmf.mappings.otherOperations.write.includes('runCommandRaw')) {
+    return '' // https://github.com/prisma/prisma/issues/8189
+  }
+
+  return `
+  /**
+   * Executes a raw mongo command and returns the result of it.
+   * @example
+   * \`\`\`
+   * const user = await (prisma as any).$runCommandRaw({
```

Comment:
```suggestion
   * const user = await prisma.$runCommandRaw({
```
`as any` is suspicious here, is it really needed?